internal/services/fabric/fabric_capacity_resource_test.go

```diff
@@ -0,0 +1,207 @@
+package fabric_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/fabric/2023-11-01/fabriccapacities"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type FabricFabricCapacityResource struct{}
+
+func TestAccFabricFabricCapacity_basic(t *testing.T) {
```

Comment:
Just noticed the naming of this, I don't think it's necessary to repeat the service name here.. 

Please rename the remaining tests
```suggestion
type FabricCapacityResource struct{}

func TestAccFabricCapacity_basic(t *testing.T) {
```

Reply:
Thanks for your feedback, fixed.